src/test/kotlin/org/rust/ide/inspections/RustInspectionsTest.kt

```diff
@@ -23,6 +23,19 @@ class RustInspectionsTest : RustTestCaseBase() {
         assertThat(openFiles.find { it.name == "foo.rs" })
             .isNotNull()
     }
+    fun testUnnecessaryParenthesis() {
+        enableInspection<UnnecessaryParenthesisInspection>()
+        myFixture.testHighlighting(true, false, true, fileName)
```

Comment:
>if i change to myFixture.testHighlighting(false, false, false, fileName) in this func https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/ide/inspections/RustInspectionsTest.kt#L33 it will still pass the test, this is really wired.

Not sure what do you mean here, but with `false, false, false` the test will not check highlighting at all, so basically any inspection will pass it